STUDIO/org.openl.security.standalone/resources/db/flyway/common/V10.3__Replace_privileges_with_permissions.sql

```diff
@@ -0,0 +1,6 @@
+DELETE FROM OpenL_Group_Authorities WHERE authority IN (SELECT apm.authority FROM acl_permission_mapping apm WHERE apm.authority <> 'VIEW_PROJECTS');
+
+DELETE FROM OpenL_Group_Authorities WHERE authority IN ('CREATE_TABLES', 'EDIT_TABLES', 'REMOVE_TABLES');
+
+DROP TABLE acl_repo_root;
+DROP TABLE acl_permission_mapping;
```

Comment:
Merge  it with the script, where `acl_repo_root` and `acl_permission_mapping` tables were created

Reply:
I can't do it. It is seperated because between these two scripts db specific script is executed. Merging breaks the idea.